gap/Parser.gi

```diff
@@ -211,7 +212,16 @@ InstallGlobalFunction( AutoDoc_Parser_ReadFiles,
                         current_line := ReadLineWithLineCount( filestream );
                         NormalizeWhitespace( current_line );
                     od;
```

Comment:
It seems that further up, you explicitly check for `KeyDependentOperation`. But then mix it with a check for `Declare*`. How about remembering the result of that check, and then using it here? That would result in more robust code, wouldn't it? As you could use that to decide whether to look for `);` or `,`, no? It would also automatically make clear what's going on.